Tran-Viet-Hoang/Addin1/ClassLibrary2/ClassLibrary2/UI/Views/MainView.xaml.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.IO;
+using Microsoft.Win32;
+
+namespace ClassLibrary2.UI.Views
+{
+    /// <summary>
+    /// Interaction logic for MainView.xaml
+    /// </summary>
+    public partial class MainView : Window
+    {
+        public MainView()
+        {
+            InitializeComponent();
+        }
+
+        private void btnload_Click(object sender, RoutedEventArgs e)
+        {
+            OpenFileDialog openFileDialog = new OpenFileDialog();
+            //openFileDialog.Filter = "Access files (*.mdb)|*.mdb";
+            if (openFileDialog.ShowDialog() == true)
+                txtinput.Text = openFileDialog.SafeFileName;
```

Comment:
it should be: .... = openFileDialog.FileName; 
then textbox will show file path